scripts/processing/01-run_anatomy.ipy

```diff
@@ -29,18 +29,18 @@ def process_subject_anat(subject_id):
 
     t1_fname = "%s/ds117/%s/anatomy/highres001.nii.gz" % (study_path, subject)
     log_fname = "%s/ds117/%s/my-recon-all.txt" % (study_path, subject)
-    !recon-all -all -s $subject -i $t1_fname > $log_fname
+    !recon-all -all -s $subject -sd $subjects_dir -i $t1_fname > $log_fname
```

Comment:
How about ``!recon-all -all -s $subject -sd $subjects_dir -i $t1_fname | tee $log_fname``?
That would also print the log.